test/distrib/php/run_distrib_test.sh

```diff
@@ -32,8 +32,9 @@ set -ex
 
 cd $(dirname $0)
 
-cp -r $EXTERNAL_GIT_ROOT/input_artifacts/grpc-php.tgz .
+cp -r $EXTERNAL_GIT_ROOT/input_artifacts/grpc-*.tgz .
```

Comment:
There is actually somewhat a clash of PHP's artifact name, in the form of grpc-1.0.0.tgz, with some of the Python's artifact name, like grpc-tools-1.1.0-dev.tgz. That's why I need that elaborate `find ... | cut ... | xargs ...` command, in the distribtest itself, to find out exactly the PHP artifact.